internal/pm/package_manager.go

```diff
@@ -42,6 +42,7 @@ type PackageInfo struct {
 	PrimaryKey         int       `json:"PrimaryKey"`
 	Hash               string    `json:"Hash"`
 	CreatorUID         string    `json:"CreatorUid"`
+	Scope              string    `json:"Scope"`
```

Comment:
this field will be visible via api, but I don't think arak can fail with it

Reply:
I tested with WinTAK and no errors were given